src/systemd-sonic-generator/ssg-test.cc

```diff
@@ -163,6 +183,21 @@ class SsgFunctionTest : public SystemdSonicGeneratorFixture {
   private:
 };
 
+
+struct SsgMainConfig {
+    int num_asics;
+    bool is_smart_switch_npu;
+    bool is_smart_switch_dpu;
+    int num_dpus;
+    SsgMainConfig() {
+        num_asics = 0;
+        is_smart_switch_npu = false;
+        is_smart_switch_dpu = false;
+        num_dpus = 0;
+    }
+};
```

Comment:
[minor]
```
struct SsgMainConfig {
    int num_asics = 0;
    bool is_smart_switch_npu = false;
    bool is_smart_switch_dpu = false;
    int num_dpus = 0;
};
```
Does the same but is more compact.

Reply:
done